tests/test_custom_type_fields_container.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+
+import conftest
+
+import ansys.dpf.core as dpf
+
```

Comment:
Can we have a test of getting the CustomTypeFC as output from an operator?

Reply:
@rafacanton, yes, it will be. I need the `Compression` plugin to be updated for that end though.